src/content/docs/tutorial-kubernetes-learn/tutorial-k8s-intro.mdx

```diff
@@ -0,0 +1,148 @@
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "200px" }}>
+        Guided install option
+      </th>
+      <th>
+        Description
+      </th>
+    </tr>
+  </thead>
+    <tbody>
+    <tr>
+      <td>
+        [Guided install](https://one.newrelic.com/nr1-core?account=2498654&state=d1aae74b-0ad6-b0f3-093d-cc89ecf89234)
+      </td>
+      <td>
+        Use this if your New Relic organization does **not** use the [EU](/docs/using-new-relic/welcome-new-relic/get-started/our-eu-us-region-data-centers) data center, and you don't need the bonus dashboards and alerts from the quickstart.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        [Guided install (EU)](https://one.eu.newrelic.com/nr1-core?account=2498654&state=d1aae74b-0ad6-b0f3-093d-cc89ecf89234)
+      </td>
+      <td>
+        Use this if your New Relic organization uses the [EU](/docs/using-new-relic/welcome-new-relic/get-started/our-eu-us-region-data-centers) data center, and you don't need the bonus dashboards and alerts from the quickstart.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        [Guided install with quickstart](https://one.newrelic.com/launcher/catalog-pack-details.launcher/?pane=eyJuZXJkbGV0SWQiOiJjYXRhbG9nLXBhY2stZGV0YWlscy5jYXRhbG9nLXBhY2stY29udGVudHMiLCJxdWlja3N0YXJ0SWQiOiI4OGE3OWY1Mi04MWMxLTRmYTItOTlmOC0zY2I1YjAxMmYxNjAifQ==)
+      </td>
+      <td>
+        Use this option if your New Relic organization does **not** use the [EU](/docs/using-new-relic/welcome-new-relic/get-started/our-eu-us-region-data-centers) data center, and you also want to install some bonus dashboards and alerts from the quickstart.
+      </td>
+    </tr>
+    </tbody>
+</table>    
+
+    </Step>
+
+<Callout variant="tip">
+Additionally, you can start monitoring Kubernetes clusters using our Auto-telemetry with Pixie. [Learn more about Auto-telemetry with Pixie here](/docs/kubernetes-pixie/auto-telemetry-pixie/get-started-auto-telemetry-pixie/).
+</Callout>
+
+<Step>
+### Instrument applications
+
+Kubernetes excels at abstracting away much of the toil associated with managing containerized workloads. That abstraction adds complexity when correlating indivdual applications to their underlying infrastructure. 
+
+<img
+    title="apm summary page"
+    alt="A screenshot depicting the APM summary page."
+    src={apmApmCompleteSummaryPage}
+/>
+
+To correlate data from applications in your Kubernetes pods to the kubernetes system as a whole, follow [the instructions in this doc](/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/link-apm-applications-kubernetes/). This process will use our application performance monitoring (APM) capabilities to scrape your applications for performance data and send them to New Relic.
```

Comment:
I think the link text should be more descriptive of what's behind the link. This helps people with vision problems who are using screen readers.

Reply:
Like New Relic, I don't think you need so many "Kubernetes". You've established the context quite strongly.

Reply:
"This process" is a little vague